docs/configuration.rst

```diff
@@ -31,6 +31,27 @@ The following is an example of this:
         },
     }
 
+EUS images
+==========
+EUS(Extended Update Support) image is a special kind of image that should be
+handled a little bit differently than common Freshmaker image. Mainly because
+it’s unpublished and Freshmaker rebuilds images that are in the published
+repository. But since those images still need to be rebuilt there is
```

Comment:
```suggestion
EUS (Extended Update Support) base images are handled a little bit differently than
standard container images. Mainly because its repository is unpublished and
Freshmaker usually rebuilds images from published repositories. Since those images
still need to be rebuilt there is
```